content/versions.yml

```diff
@@ -23,42 +26,22 @@
 - title: '16.9'
   changelog: https://github.com/facebook/react/blob/main/CHANGELOG.md#1690-august-8-2019
 - title: '16.8'
-  path: /version/16.8
   changelog: https://github.com/facebook/react/blob/main/CHANGELOG.md#1680-february-6-2019
-  url: https://5d4b5feba32acd0008d0df98--reactjs.netlify.com/
```

Comment:
@acdlite curious to know why you removed these Netlify permalinks.

Did they go offline without notice? I'm facing this problem on the Docusaurus versioning page: https://github.com/facebook/docusaurus/issues/8634